### DISCUSSION TITLE
Aula - Comentarios

### DISCUSSION THREAD

**Discussion by cmsouzaac** [balta-io/2801#93] [Perguntas e Respostas] [answered]
@andrebaltieri 

Os comentarios do tipo 
```
/// <summary>
/// Descrição
/// </summary>
```

é a mesma coisa como se eu estivesse comentando meu codigo? tipo eu posso colocar exemplos e anotações dentro dessa estrutura?

**Comment by andrebaltieri** [comment, marked as answer]
@cmsoouza sim! Como eu mostro na aula de comentários do curso de C#, este tipo de comentário em diversas linhas também é considerado um comentário..

Porém, este tem um efeito especial, o Visual Studio entende ele e mostra estas informações na hora que você vai consumir o método.

**Reply by cmsouzaac** [reply]
Obg professor @andrebaltieri
